qa/crontab/teuthology-cronjobs

```diff
@@ -36,20 +36,21 @@ CEPH_QA_EMAIL="ceph-qa@ceph.io"
 0 8  * * 5 CEPH_BRANCH=octopus; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -m $MACHINE_NAME -s smoke -k distro -e $CEPH_QA_EMAIL -p 70
 7 8  * * 6 CEPH_BRANCH=pacific; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -m $MACHINE_NAME -s smoke -k distro -e $CEPH_QA_EMAIL -p 70
 
-## master branch runs - weekly
+## quincy branch runs - weekly
```

Comment:
I think master runs are still useful for a baseline. Can we make them at least monthly, maybe with a higher priority too?

Also, for new releases we usually run ~2 times a week, which I think is appropriate. Let's do that too?

Reply:
the last manual master baseline i saw was from feb 2nd in 
http://pulpito.ceph.com/?branch=wip-yuriw-master-2.1.22

i agree with @batrick, i'd prefer to find master runs in pulpito if we have the resources for it

Reply:
Is there a reason the `master` branch name can't be used in these baseline runs? A `-S` can be used for a specific revision. Also a `--suite-branch` with the same revision so they are in sync. (You can't specify the revision for the `--suite-branch` like you can for `--ceph`.)

Reply:
It can absolutely be used, @yuriw can you please keep this in mind.

Reply:
sounds like using the master branch name for the weekly baseline runs would give us the coverage and discoverability we're looking for